### PR TITLE
Interval quarter

### LINKED ISSUE #1252: Add support for an interval with a quarter period
cc @kszucs

### PR DESCRIPTION
**Pull request by kszucs**
- quarter support
- allowed conversion between year - quarter - month
- named test cases

resolves #1252

### CODE REVIEW

File: ci/datamgr.py [hunk 1]
```diff
@@ -186,8 +186,8 @@ def sqlite(script, tables, database, data_directory):
         df = pd.read_csv(path, index_col=None, header=0)
         with engine.begin() as con:
             df.to_sql(table, con, index=False, if_exists='append')
-    engine.execute('VACUUM')
-    engine.execute('VACUUM ANALYZE')
+    # engine.execute('VACUUM')
+    # engine.execute('VACUUM ANALYZE')
```
Comment (cpcloud) [inline ci/datamgr.py]: It's fine to kill these lines.

**Comment by cpcloud** [comment]
@kszucs Can you add a release note to `docs/source/release.rst`? Then will merge.

**Comment by cpcloud** [comment]
@kszucs can you rebase and resolve the tiny doc conflict? i will merge after that

**Comment by kszucs** [comment]
@cpcloud done

**Pull request closed without merging**